tests/demonstrator_scripts/test_column_simulation.py

```diff
@@ -29,7 +29,7 @@ def test_column_simulation():
 
     # temperature setings:
     parameters['T_0'] = 40  # initial temperature of concrete
-    parameters['T_boundary'] = 40  # constant boundary temperature
+    parameters['T_bc1'] = 20  # constant boundary temperature
```

Comment:
what is bc1 (I guess there is another bc2?

Reply:
this goes back to more complex test cases with different temperatures at different positions. this is however coded in fenics_concrete, so not something we would change here.